dev/cli.go

```diff
@@ -15,7 +15,13 @@ type CLI struct {
 }
 
 // Build the CLI binary
-func (cli *CLI) File(
+// +deprecated
+func (cli *CLI) File(ctx context.Context, platform dagger.Platform) (*dagger.File, error) {
+	return cli.Binary(ctx, platform)
+}
```

Comment:
Couple points:
- This removes the optionalness of `platform`, so callers will still likely break
- `// +deprecated` isn't a pragma with a special value, I don't think we should use it like this (if we want it to be a thing, we can add it)

Reply:
> This removes the optionalness of platform, so callers will still likely break

That's an oversight, thanks for catching.